apps/api/src/app/environments/dtos/environment-response.dto.ts

```diff
@@ -15,11 +13,14 @@ export class EnvironmentResponseDto {
   @ApiProperty()
   identifier: string;
 
-  @ApiProperty({
-    type: [ApiKey],
-  })
-  apiKeys: IApiKey[];
+  @ApiPropertyOptional()
+  apiKeys?: IApiKeyDto[];
```

Comment:
Optional because get-environment.usecase does not return the api keys, i wonder if we need to return them at all under environment-response.dto. I would suggest removing it at altogether and providing the keys only under GET `/environments/api-keys`

Reply:
The `GET /environments` does use that interface and returns the `apiKeys`.

Reply:
True, that is why I left it as optional because `GET /environments/me` do not return it.